packages/expo/src/hooks/useOAuth.ts

```diff
@@ -101,6 +102,8 @@ export function useOAuth(useOAuthParams: UseOAuthFlowParams) {
       createdSessionId = signUp.createdSessionId || '';
     }
 
+    Linking.openURL(url);
```

Comment:
We shouldn't trigger the deep linking if there are verification errors, missing factors, such as MFA

Reply:
Ideally the `action_complete_redirect_url` would be nice here, cause then this would be the one FAPI would redirect to if the SignIn object is complete